spec/models/ability_spec.rb

```diff
@@ -2,36 +2,35 @@
 require 'cancan/matchers'
 
 describe Ability do
-  before(:each) do
-    @member = FactoryGirl.create(:member)
-    @ability = Ability.new(@member)
-  end
+
+  let(:member) { FactoryGirl.create(:member) }
+  let(:ability) { Ability.new(member) }
 
   context "notifications" do
     it 'member can view their own notifications' do
-      @notification = FactoryGirl.create(:notification, :recipient => @member)
-      @ability.should be_able_to(:read, @notification)
+      @notification = FactoryGirl.create(:notification, :recipient => member)
```

Comment:
@pozorvlak just to confirm, when you say "make @notification a local variable", is it the same as using let for the notification?